elasticsearch.md

```diff
@@ -1752,22 +1751,59 @@ And ReadonlyREST ES will load "S3cr3tP4ss" as `bind_password`.
 One of the neatest feature in ReadonlyREST is that you can use dynamic variables inside most rules values. The variables you can currently replace into rules values are these:
 
 * `@{acl:user}` gets replaced with the username of the successfully authenticated user. Using this variable is allowed only in blocks where one of the rules is authentication rule \(of course it must be rule different from the one containing given variable\).
-* `@{user}` old style user variable definition. Preferred approach is to use `@{acl:user}`.
 * `@{acl:current_group}` gets replaced with user's current group. Usually resolved by authorization rule defined in block, but value can be also retrieved by means of kibana plugin. This variable doesn't specify usage requirements.
-* `@{xyz}` gets replaced with any `xyz` HTTP header included in the incoming request \(useful when reverse proxies handle authentication\)
+* `@{acl:available_groups}` gets replaced with available groups found in the authorization rule. 
+* `@{header:<header_name>}` gets replaced with the header value of the HTTP header name `<header_name>` included in the incoming request \(useful when reverse proxies handle authentication\)
+* `@{jwt:<json_path>}` get replaced with value (or values) found in JWT under the given JSON path
 
 #### Indices from user name
 
 You can let users authenticate externally, i.e. via LDAP, and use their user name string inside the `indices` rule.
 
-```text
+```yaml
 readonlyrest:
     access_control_rules:
 
-    - name: "Users can see only their logstash indices i.e. alice can see alice_logstash-20170922"
+    - name: "Users can see only their logstash indices" # i.e. alice can see alice_logstash-20170922
       ldap_authentication:
         name: "myLDAP"
-      indices: ["@{acl:user}_logstash-*"]
+      indices: ["@{acl:user}_logstash-*"] 
+
+    # LDAP connector settings omitted, see LDAP section below..
+```
+
+#### Indices from available groups
+
+You can let users authorize externally, i.e. via LDAP, and use their group names strings inside the `indices` rule.
+
+```yaml
+readonlyrest:
+    access_control_rules:
+
+    - name: "Users can see only logstash indices for their departments" # i.e. alice belongs to 'dev' and 'ops' department groups, so she can see dev_logstash-20170922, ops_logstash-20170922
+      ldap_auth:
+        name: "myLDAP"
+        groups: ["dev", "ops", "qa"]
+      indices: ["@explode{acl:available_groups}_logstash-*"] #  notice that 'available_groups' is an array. `indices` rule has multi-value context (we can configure an array of index names in it). It means that we have to create several indices names from the available_groups array and the optional static part (here: _logstash-*) - it can be achieved with "explode" keyword i.e from available_groups=[dev, ops] we will get: indices: ["dev_logstash-*", "ops_logstash-*"] 
```

Comment:
woa this in-comment explanation has a ton of detail! How about we create a small paragraph with a few examples, and in the comment we put the "expanded" conf?